clown/common/.gitignore

```diff
@@ -41,10 +41,6 @@ node_modules/
 # Builds
 dist
 
-# Editor configs
-.vscode/**/*
-!.vscode/launch.json
-
```

Comment:
Should these kinds of changes be considered "breaking"? It would cause all downstream dependencies to fail `clown check` if they upgrade.